src/librustdoc/json/mod.rs

```diff
@@ -161,9 +162,8 @@ impl<'tcx> FormatRenderer<'tcx> for JsonRenderer<'tcx> {
         ))
     }
 
-    fn make_child_renderer(&self) -> Self {
-        self.clone()
-    }
+    fn make_child_renderer(&mut self) -> Self::InfoType {}
```

Comment:
NIT: These should panic, as "child renderer" doesn't make sense in JSON, so something's gone wrong if they're being called.

Reply:
I'm surprised since it's a common code path for all renderers.

Reply:
It only (should) happens if RUN_ON_MODULE is true, which it isn't for json.

Reply:
Oh I see. Completely missed that!